apps/frontend/client/.env.example

```
@@ -14,6 +14,10 @@ PUBLIC_LOG_LEVEL=
 PUBLIC_LOG_PERSIST_LEVEL=
 PUBLIC_PARSE_LEVEL=safe
 PUBLIC_SITE_URL=
+# client is static Firebase Hosting with no server of its own — browser logs
+# forward cross-origin to hub's /api/internal_logging in staging/production.
+# Leave empty in emulator/dev (uses the relative-path dev sink instead).
+PUBLIC_LOG_ENDPOINT=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PUBLIC_LOG_ENDPOINT key should go before the PUBLIC_LOG_LEVEL key

(UnorderedKey)